learn/index.html

```diff
@@ -154,6 +154,14 @@
 
       <ul class="cards">
         <li>
+          <h3>Lunch &amp; Learn</h3>
+          <p>"Better know a language: PureScript" by <a href="http://taylor.fausak.me">Taylor Fausak</a></p>
```

Comment:
If you're making another edit anyway, could you make these quotes match the others please?